src/views/migrationpolicies/details/tabs/details/components/MigrationPolicyDetailsSection/components/MigrationPolicyDescriptionItem.tsx

```diff
@@ -1,25 +1,46 @@
 import React from 'react';
 
+import { DescriptionItemHeader } from '@kubevirt-utils/components/DescriptionItem/DescriptionItemHeader';
 import { NO_DATA_DASH } from '@kubevirt-utils/resources/vm/utils/constants';
 import {
   DescriptionListDescription,
   DescriptionListGroup,
   DescriptionListTermHelpText,
-  DescriptionListTermHelpTextButton,
+  Flex,
+  FlexItem,
 } from '@patternfly/react-core';
 
 type MigrationPolicyDescriptionItemProps = {
   title: string;
-  description: React.ReactNode;
+  description?: React.ReactNode;
+  isPopover?: boolean;
+  popoverContent?: React.ReactNode;
+  moreInfoURL?: string;
+  breadcrumb?: string;
+  hasAutoWidth?: boolean;
 };
 
 const MigrationPolicyDescriptionItem: React.FC<MigrationPolicyDescriptionItemProps> = ({
   title,
   description,
+  isPopover,
+  popoverContent,
+  moreInfoURL,
+  breadcrumb,
 }) => (
   <DescriptionListGroup>
     <DescriptionListTermHelpText>
-      <DescriptionListTermHelpTextButton>{title}</DescriptionListTermHelpTextButton>
+      <Flex>
```

Comment:
Do we need the flex and flex item here? if all the rest of the components don't use it?

Reply:
I can see the components use it, when using `Badge`. I tried to omit flex and found that with omitting it, we achieve a bad position of the Badge in the UI. it is not because of that specific component there, but because aligning more things in one row while being those things in one column.

Reply:
No, we don't. Removed. I used that earlier to properly style the popovers (aligning mostly). Now achieved by css. Thanks!